From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ignacio Zamora <nachopitt@gmail.com>
Date: Sat, 8 Oct 2022 09:50:56 -0500
Subject: Fix typo in CONFIG_USBNET_DEVADDR

Fix typo that was caused by the same feature being split in to 2 different
configuration options. Replace CONFIG_USBNET_DEVADDR with
CONFIG_USBNET_DEV_ADDR

Signed-off-by: Ignacio Zamora <nachopitt@gmail.com>
---
 drivers/usb/gadget/Kconfig | 2 +-
 drivers/usb/gadget/ether.c | 2 +-
 include/configs/warp7.h    | 2 --
 3 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/drivers/usb/gadget/Kconfig b/drivers/usb/gadget/Kconfig
index 111111111111..222222222222 100644
--- a/drivers/usb/gadget/Kconfig
+++ b/drivers/usb/gadget/Kconfig
@@ -241,7 +241,7 @@ config USB_ETH_RNDIS
 
 endchoice
 
-config USBNET_DEVADDR
+config USBNET_DEV_ADDR
 	string "USB Gadget Ethernet device mac address"
 	default "de:ad:be:ef:00:01"
 	help
diff --git a/drivers/usb/gadget/ether.c b/drivers/usb/gadget/ether.c
index 111111111111..222222222222 100644
--- a/drivers/usb/gadget/ether.c
+++ b/drivers/usb/gadget/ether.c
@@ -2620,7 +2620,7 @@ static int usb_eth_probe(struct udevice *dev)
 	priv->netdev = dev;
 	l_priv = priv;
 
-	get_ether_addr(CONFIG_USBNET_DEVADDR, pdata->enetaddr);
+	get_ether_addr(CONFIG_USBNET_DEV_ADDR, pdata->enetaddr);
 	eth_env_set_enetaddr("usbnet_devaddr", pdata->enetaddr);
 
 	return 0;
diff --git a/include/configs/warp7.h b/include/configs/warp7.h
index 111111111111..222222222222 100644
--- a/include/configs/warp7.h
+++ b/include/configs/warp7.h
@@ -100,8 +100,6 @@
 /* USB Device Firmware Update support */
 #define DFU_DEFAULT_POLL_TIMEOUT	300
 
-#define CONFIG_USBNET_DEV_ADDR		"de:ad:be:af:00:01"
-
 /* Environment variable name to represent HAB enable state */
 #define HAB_ENABLED_ENVNAME		"hab_enabled"
 
-- 
Armbian

